doc/instance-exec.md

```diff
@@ -1,57 +1,126 @@
-# Instance command execution
+---
+discourse: 9223
+---
 
-LXD makes it easy to run a command inside a given instance.
+(run-commands)=
+# How to run commands in an instance
+
+LXD makes it easy to run a command inside an instance.
```

Comment:
Similar to the file access feature, we should probably indicate why this may be preferable than using, say, ssh.

> You can run a command inside an instance using the LXD client without needing to access the instance via the network.